frontend/src/features/public-form/utils/createSubmission.ts

```diff
@@ -180,6 +189,55 @@ export const createClearSubmissionWithVirusScanningFormData = (
   return formData
 }
 
+/**
+ * Used for MRF submissions v3 (after virus scanning).
+ * @returns formData containing form responses and attachments.
+ * @throws Error if form inputs are invalid or contain malicious attachment(s).
+ */
+export const createClearSubmissionWithVirusScanningFormDataV3 = (
+  formDataArgs: CreateMultirespondentSubmissionFormDataArgs,
+  fieldIdToQuarantineKeyMap: FieldIdToQuarantineKeyType[],
+) => {
+  const { formFields, formInputs, ...formDataArgsRest } = formDataArgs
+
```

Comment:
q: missed this ytd, do we perform virus scanning for mrf attachments?

Reply:
yes we do! It's assumed that encryption boundary shift + virus scanning is 100%